dataprofiler/profilers/profiler_options.py

```diff
@@ -991,15 +1000,37 @@ def _validate_helper(self, variable_path='StructuredOptions'):
             ('data_labeler', DataLabelerOptions),
             ('correlation', CorrelationOptions)
         ])
-
-        for column in self.properties:
+        properties = self.properties
+        properties.pop('null_values')
+        for column in (properties):
             if not isinstance(self.properties[column], prop_check[column]):
                 errors.append("{}.{} must be a(n) {}.".format(
                     variable_path, column, prop_check[column].__name__))
             else:
                 errors += self.properties[column]._validate_helper(
                     variable_path=(variable_path + '.' + column
                                    if variable_path else column))
+        if self.null_values:
+            import re
```

Comment:
imports should be at the top of the file in this case, we would only put it here if it was expensive to import and only this function needed it